pkg/cosign/verify.go

```diff
@@ -484,6 +484,12 @@ func VerifyImageSignatures(ctx context.Context, signedImgRef name.Reference, co
 	// entity that minimizes registry requests when supplied with a digest input
 	digest, err := ociremote.ResolveDigest(signedImgRef, co.RegistryClientOpts...)
 	if err != nil {
+		if strings.Contains(err.Error(), "MANIFEST_UNKNOWN") {
+			return nil, false, &VerificationError{
```

Comment:
Nit: I think best practice for wrapping errors is to store the inner error inside the struct and format the message inside the Error function.

See: https://go.dev/blog/go1.13-errors